activities.min.js

```diff
@@ -0,0 +1,40 @@
+(function(){/*
+
+ Copyright 2017 The Web Activities Authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS-IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
```

Comment:
Should we also add version number for *min.js?

Reply:
There are bunch of versions in there already. But putting it in the header is a bit hard since it's created by closure compiler. I did a couple of tries - no luck so far. Let's try later.